modules/core/src/main/scala/dev/profunktor/redis4cats/effect/TxExecutor.scala

```diff
@@ -29,26 +29,30 @@ import scala.util.control.NonFatal
 
 import cats.effect.kernel._
 import cats.syntax.all._
+import cats.effect.std.Dispatcher
 
 private[redis4cats] trait TxExecutor[F[_]] {
   def delay[A](thunk: => A): F[A]
   def eval[A](fa: F[A]): F[A]
   def start[A](fa: F[A]): F[Fiber[F, Throwable, A]]
-  def liftK[G[_]: Async]: TxExecutor[G]
+  def liftK[G[_]: Async: Dispatcher]: TxExecutor[G]
```

Comment:
this would be a breaking change as it adds additional parameters.
I dont think TxExecutor is the right place for this, it is meant for [Redis transactions ](https://redis.io/docs/latest/develop/interact/transactions/)

Reply:
I would add the `Dispatcher` is not meant to be used as a typeclass, i.e. should be passed explicitly